README.md

```diff
@@ -142,16 +142,58 @@ We are also working on the following features:
 - CloudRun Services
 - Test Automation and Ci/CD with CloudBuild
 
+### Microservices locally using Minikube
```

Comment:
The README.md is for the template project. For the detailed deployment README, can you add to https://github.com/GoogleCloudPlatform/solutions-template/blob/main/build_tools/README.md?

Let me know if the two separate READMEs make sense.